membership_variable_period/models/account_move_line.py

```diff
@@ -12,20 +12,29 @@
 class AccountMoveLine(models.Model):
     _inherit = "account.move.line"
 
-    def _prepare_membership_line(self, move, product, price_unit, line_id, qty=1.0):
+    def _prepare_membership_line(self, move, product, price_unit, line, qty=1.0):
```

Comment:
I'm not against changing this arg type, but doing so in stable code could break someone else extensions...

It also makes the commit meaningful content more difficult to follow.

Reply:
Done